doc/man7/ossl-guide-migration.pod

```diff
@@ -157,6 +157,13 @@ To ensure the future compatibility, the engines should be turned to providers.
 To prefer the provider-based hardware offload, you can specify the default
 properties to prefer your provider.
 
+Setting engine-based default low-level crypto method such as B<RSA_METHOD> or
+B<EC_KEY_METHOD> is still possible and keys inside the default provider will
+use the engine-based implementation for the crypto operations. However
```

Comment:
Not only engines - e.g. OpenSSH implements custom RSA/EC methods for PKCS#11 keys.